src/lib/serviceRoutinesV2/getAllSubscriptions.cpp

```diff
@@ -54,8 +55,10 @@ std::string getAllSubscriptions
 
   std::vector<ngsiv2::Subscription> subs;
   OrionError                        oe;
+  int                               offset = atoi(ciP->uriParam[URI_PARAM_PAGINATION_OFFSET].c_str());
+  int                               limit  = atoi(ciP->uriParam[URI_PARAM_PAGINATION_LIMIT].c_str());
 
```

Comment:
We are using the same approach in all the others paginated operations (e.g. mongoQueryContext()) from time ago, so I guess that the default have set in some common place before (parsing routines? servicePaths?). However, this theory should be confirmed, just to be sure.